src/logic/Window.swift

```diff
@@ -116,6 +116,12 @@ class Window {
 
     func minDemin() {
         if isWindowlessApp { return }
+        guard !isTabbed else {
+            if let tabHoldingWindows = application.tabHoldingWindows[spaceId], tabHoldingWindows.count == 1 {
+                tabHoldingWindows.first!.minDemin()
+            }
+            return
+        }
```

Comment:
I think it would be best to hide the buttons, and not try to do things that the OS itself doesn't allow the user to do. This is in line with our previous discussions here: https://github.com/lwouis/alt-tab-macos/pull/2021#issuecomment-1284711792

Reply:
It seems to me that AltTab already does things not allowed by the OS, depending on your definition. My issue with this though is that it only covers one case.

Reply:
Yes, I meant that if we can't have great support for some approach, it's probably better to give up on it, and it shouldn't be too missed by users since they are used to not having such capabilities in the first place, from the native OS UI.